babel.config.js

```diff
@@ -41,7 +41,7 @@ const metro = {
  * By default <React.Profiler> is disabled in production as it adds small overhead
  * When CAPTURE_METRICS is set we're explicitly saying that we want to capture metrics
  * To enable the <Profiler> for release builds we add these aliases */
-if (process.env.CAPTURE_METRICS) {
+if (process.env.CAPTURE_METRICS === 'true') {
```

Comment:
`process.env` values are `string` so we need to check explicitly for `"true"` 
That how other compile check are made as well (e.g. `process.env.ANALYZE_BUNDLE === 'true'`)